atc/db/check.go

```diff
@@ -11,6 +12,7 @@ import (
 	"github.com/concourse/concourse/atc"
 	"github.com/concourse/concourse/atc/db/lock"
 	"github.com/lib/pq"
+	"go.opentelemetry.io/otel/api/propagators"
```

Comment:
We probably don't want to use opentelemetry directly? It looks like the `tracing` package was designed to wrap this dependency.

Reply:
Just doing a quick search it actually looks like the scheduler package uses it directly, so maybe we're not being strict about this anymore.

Reply:
yeah i hoped to make a clean adapter for that, just haven't gotten around to it yet.

Reply:
@clarafu I think we can just change `propagators.Supplier` to `SpanContext` to get rid of this dependency?

Reply:
@pivotal-jwinters I think we need to keep it as `propagators.Supplier` because it needs to match the interface for WithSpanContext https://github.com/concourse/concourse/blob/b38487edf2d69b1fe8d75ed19b4eba566fd7ed46/tracing/tracer.go#L90-L92

as we pass call `StartSpanFollowing` in lidar checker https://github.com/concourse/concourse/blob/b38487edf2d69b1fe8d75ed19b4eba566fd7ed46/atc/lidar/checker.go#L52-L60
using that span context.

Reply:
cool 👍